### PR TITLE
add partial for version banner

### PR DESCRIPTION
**Pull request by moficodes**
- add partial for version banner
- add a version called latest in the version list

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -103,10 +103,17 @@ params:
...
   latest_stable_version: v3.5
+  version:
+
+    deprecation_warning: >-
+      This version of etcd is no longer supported. For the latest version, please see the [latest stable version](/docs/latest/).
   versions:
     - version: v3.6
       url: /docs/v3.6/
 
+    - version: latest
```
Comment (jmhbnz) [inline config.yaml:114]: Hey @moficodes thanks for tackling this.  The deploy preview is looking good, however can you explain why we need this new version entry added?

I thought `docs/latest` and `docs/next` where meta versions that we manage as redirects in https://github.com/etcd-io/website/blob/bdb510e276299794734e08ddd96f0cb2686bbc59/layouts/index.redirects#L1-L12

Comment (moficodes) [reply config.yaml]: A local change I fotgot to remove.
Removed now!

Comment (jmhbnz) [reply config.yaml]: Hmm one other issue, unfortunately the banner is showing on 3.4 https://deploy-preview-900--etcd.netlify.app/docs/v3.4/

3.4 is definitely still supported so we need to tweak the logic in the partial to address I think

**Comment by moficodes** [comment]
@jmhbnz

**Comment by jmhbnz** [comment]
I am still seeing that v3.4 is showing up in the deploy preview as having the deprecation banner? https://deploy-preview-900--etcd.netlify.app/docs/v3.4/

v3.4 is still supported so should not have the banner, please let me know if I am missing something.

**Comment by ivanvc** [comment]
This change looks good to me. One thing worth noting is that the v3.6 docs now show a warning, too. I think it should be fine, but I just wanted to raise awareness of it.

https://deploy-preview-900--etcd.netlify.app/docs/v3.6/

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @moficodes

**Review by ivanvc — APPROVED** [review]
LGTM. Thanks, Mofi.

**Pull request merged**